packages/Webkul/Admin/src/Http/Controllers/Settings/Tax/TaxRateController.php

```diff
@@ -176,28 +176,25 @@ public function destroy($id): JsonResponse
     /**
      * Import function for the upload.
      *
-     * @return \Illuminate\Http\Response
+     * @return \Illuminate\Http\JsonResponse
      */
     public function import()
     {
         $valid_extension = ['xlsx', 'csv', 'xls'];
 
         if (! in_array(request()->file('file')->getClientOriginalExtension(), $valid_extension)) {
-            session()->flash('error', trans('admin::app.export.upload-error'));
+            return new JsonResponse([
+                'message' => trans('admin::app.settings.taxes.rates.index.import.upload-error'),
+            ], 500);
         } else {
             try {
                 $excelData = (new DataGridImport)->toArray(request()->file('file'));
 
                 foreach ($excelData as $data) {
                     foreach ($data as $column => $uploadData) {
-                        if (is_null($uploadData['state'])) {
-                            $uploadData['state'] = '';
-                        }
+                        $uploadData['state'] = $uploadData['state'] ?? '';
 
-                        if (
-                            ! is_null($uploadData['zip_from'])
-                            && ! is_null($uploadData['zip_to'])
-                        ) {
+                        if (! is_null($uploadData['zip_from']) && ! is_null($uploadData['zip_to'])) {
```

Comment:
Break the properties into multiple lines.

Reply:
fixed